gradle.properties

```
@@ -14,4 +14,4 @@ dokkaVersion=2.0.0
 
 org.gradle.jvmargs=-Xmx6g -XX:MaxMetaspaceSize=2048m
 org.gradle.workers.max=10
-version=1.5.1-fix-machtingdienst-SNAPSHOT
+version=1.5.1-fix-zaken-api-query-SNAPSHOT
```

let's add the pr number to the tag.
```suggestion
version=1.5.1-407-fix-zaken-api-query-SNAPSHOT
```